core/templates/pages/library-page/library-page.directive.ts

```diff
@@ -22,7 +22,7 @@ require(
 require('components/summary-tile/exploration-summary-tile.directive.ts');
 require('components/summary-tile/collection-summary-tile.directive.ts');
 require('pages/library-page/search-results/search-results.directive.ts');
-
+require('domain/classroom/classroom-backend-api.service');
```

Comment:
Add newline between service and directive imports

Reply:
Addressed.